packages/table/src/table.tsx

```diff
@@ -467,22 +466,22 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     private resizeSensorDetach: () => void;
 
     private refHandlers = {
-        cellContainer: (ref: HTMLElement) => (this.cellContainerElement = ref),
-        columnHeader: (ref: HTMLElement) => (this.columnHeaderElement = ref),
-        mainQuadrant: (ref: HTMLElement) => (this.mainQuadrantElement = ref),
-        quadrantStack: (ref: TableQuadrantStack) => (this.quadrantStackInstance = ref),
-        rootTable: (ref: HTMLElement) => (this.rootTableElement = ref),
-        rowHeader: (ref: HTMLElement) => (this.rowHeaderElement = ref),
-        scrollContainer: (ref: HTMLElement) => (this.scrollContainerElement = ref),
+        cellContainer: (ref: HTMLElement | null) => (this.cellContainerElement = ref),
+        columnHeader: (ref: HTMLElement | null) => (this.columnHeaderElement = ref),
+        mainQuadrant: (ref: HTMLElement | null) => (this.mainQuadrantElement = ref),
+        quadrantStack: (ref: TableQuadrantStack | null) => (this.quadrantStackInstance = ref),
+        rootTable: (ref: HTMLElement | null) => (this.rootTableElement = ref),
+        rowHeader: (ref: HTMLElement | null) => (this.rowHeaderElement = ref),
+        scrollContainer: (ref: HTMLElement | null) => (this.scrollContainerElement = ref),
```

Comment:
IIRC the code won't even invoke these ref handlers if we don't show the associated component in the UI, so I don't think these `| null`s are helpful in the first place.

Reply:
just adding the correct typings. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L77

I reverted tho